docs/merge_points/merge-tests.md

```diff
@@ -25,8 +25,8 @@ In this example we add an action group that modifies the original test to intera
         <deleteData createDataKey="createPreReqCategory" stepKey="deletePreReqCategory"/>
     </after>
 
-    <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin1"/>
-    <actionGroup ref="FillAdminSimpleProductForm" stepKey="fillProductFieldsInAdmin">
+    <actionGroup ref="AdminLoginActionGroup" stepKey="AdminLoginActionGroup1"/>
```

Comment:
does step key need to be lowerCamelCase?

Reply:
good catch! fixed